spec/unit/show-hide-content.spec.js

```diff
@@ -0,0 +1,245 @@
+describe('show-hide-content', function () {
+  'use strict'
+
+  beforeEach(function () {
+
+    // Sample markup
+    this.$content = $(
+
+      // Radio buttons (yes/no)
+      '<form class="js-toggle-showhide">' +
```

Comment:
Is this `js-toggle-showhide` class needed? I can't see it used in the tests or the module code.